docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -1,105 +1,354 @@
+    text-decoration: none;
+    line-height: 120.3%;
+
+    @include grid-media-min('md') {
+      line-height: 128%;
+    }
+
+    @include grid-media-min('lg') {
+      .js & {
+        padding: hb-calculate-rems(10px) 0 hb-calculate-rems(20px);
+        line-height: 110%;
+      }
+    }
+
+    .hb-main-nav__item--parent > & {
+      padding: hb-calculate-rems(21px) hb-calculate-rems(48px) hb-calculate-rems(21px) hb-calculate-rems(17px);
+
+      @include grid-media-min('lg') {
+        .js & {
+          padding: hb-calculate-rems(10px) hb-calculate-rems(20px) hb-calculate-rems(20px) 0;
+        }
+      }
+    }
+
+    &:hover:not(.is-active) {
+      background-color: $hb-menu-link-background-hover;
+    }
+
+    // Targeting only the first level of navigation on desktop
+    .hb-main-nav__menu-lv1 > li > & {
+      @include grid-media-min('lg') {
+        .js &::before {
+          content: '';
+          display: block;
+          position: absolute;
+          width: 100%;
+          height: hb-calculate-rems(4px);
+          bottom: hb-calculate-rems(8px);
+          top: auto;
+          left: 0;
+          background: transparent;
+        }
+
+        .js &.is-active {
+          background-color: transparent;
+        }
+
+        .js &.hb-main-nav__link--active-trail::before {
+          @include hb-colorful {
+            background: hb-colorful-variation(secondary);
+          }
+        }
+
+        .js &:hover {
+          background-color: transparent;
+
+          &:not(.is-active):not(.hb-main-nav__link--active-trail)::before {
+            background: $hb-menu-link-border-hover;
+          }
+        }
+      }
+    }
+
+    .hb-main-nav__menu-lv2 & {
+      @include grid-media-min('lg') {
+        .js & {
+          font-size: hb-calculate-rems(16px);
+          padding: hb-calculate-rems(10px) hb-calculate-rems(14px);
+        }
+      }
+    }
+
+    &--active-trail {
+      &::before {
+        content: '';
+        display: block;
+        width: hb-calculate-rems(5px);
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+
+        background: $hb-menu-link-border-active;
+      }
+
+      &.is-active {
+        @include hb-colorful {
+          background: rgba(darken(hb-colorful-variation(secondary), 2%), 0.2);
+
+          &::before {
+            background: hb-colorful-variation(secondary);
+          }
+        }
+      }
+    }
+
+    &.hb-nested-toggler {
+      position: relative;
+
+      &::after {
+        @include hb-menu-large-icon;
+        @include hb-menu-plus-icon;
+        content: '';
+        display: block;
+
+        position: absolute;
+        top: hb-calculate-rems(8px);
+        right: 0;
+
+        .no-js & {
+          display: none;
+        }
+
+        @include grid-media-min('md') {
+          top: hb-calculate-rems(16px);
+        }
+
+        @include grid-media-min('lg') {
+          .js & {
+            @include hb-menu-small-icon;
```

Comment:
this is counterintuitive... on large screens it goes to the small icon?? lol wut. maybe we can rename to desktop icon?

Reply:
This was refactored into a more reusable mixin.

Reply:
yay!